src/main/java/io/cdap/plugin/gcp/common/GCPUtils.java

```diff
@@ -101,14 +101,19 @@ public static String extractPrivateKey(ServiceAccountCredentials credentials) {
                                                            String serviceAccountType,
                                                            String... keyPrefix) throws IOException {
     Map<String, String> properties = new HashMap<>();
+    if (serviceAccountType == null) {
```

Comment:
Why is this change added? why it was not needed before?

Reply:
It was added to handle when serviceAccountType is macro, and this method is called in configuration stage.